Services/CO.CDP.DataSharing.WebApi.Tests/AutoMapper/DataSharingProfileTest.cs

```diff
@@ -5,6 +5,6 @@ public class DataSharingProfileTest(AutoMapperFixture mapperFixture) : IClassFix
     [Fact]
     public void ConfigurationIsValid()
     {
-        mapperFixture.Configuration.AssertConfigurationIsValid();
+        mapperFixture.Configuration?.AssertConfigurationIsValid();
```

Comment:
Seems wrong. If the Configuration is null, the test should fail. Otherwise, there's no point of it.

Reply:
That's true, makes no point as won't happen as it would actually fail much earlier if unable to load the consumed setting.
However, made it this way only to satisfy GitHub complaints, so opened for any suggestions.